contracts/HubPool.sol

```diff
@@ -398,6 +400,9 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
             "Bad Proof"
         );
 
+        // Before interacting with a particular chain's adapter, ensure that the adapter is set.
+        require(address(crossChainContracts[poolRebalanceLeaf.chainId].adapter) != address(0), "No adapter for chain");
```

Comment:
is there any advantage to prevent setting the adapter contract address to 0x0 in the `setCrossChainContracts` func?

Reply:
Actually, I think we should actually allow this to allow a chain to be de-whitelisted.